docs/pbc/all/payment-service-providers/payone/integrate-payone.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Payone
+description: Learn how you can integrate the Payone app into your Spryker shop
+template: howto-guide-template
+---
+
+To integrate Payone, follow these steps:
+
+1. In your store's Back Office, go to **Apps&nbsp;<span aria-label="and then">></span> Catalog**.
+2. Click **Payone**.
+   This takes you to the Payone app details page.
+3. In the top right corner of the Payone app details page, click **Connect app**.
+   This displays a message about the successful connection of the app to your SCCOS. The Payone app's status changes to *Connection pending*.   
+4. Go to [Payone](https://www.payone.com?ref=spryker-documentation) and obtain the credentials.
+   
+   {% info_block infoBox "Info" %}
+
+   It takes some time to obtain credentials from Payone because you have to go through a thorough vetting process by Payone, such as the "know your customer" (KYC) process before Payone verifies you.
+
+   {% endinfo_block %}
+
+5. Go back to your store's Back Office, to the Payone app details page.
+6. In the top right corner of the Payone app details page, click **Configure**.
+7. On the Payone app details page, fill in fields in the **Credentials** section.
+   ![payone-app-detais](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-app-details.png)
+8. Select *Payone Environment Mode*: **TEST** or **LIVE**.
```

Comment:
`Payone Environment Mode` as a UI element should be in bold. Also, I don't see a screenshot, so make sure to reflect the UI's case.

Reply:
Usually, you don't need to list the options. If you do, there's a reason to believe readers will not understand what the options mean. Then, you need to explain all of them. However, in this case, the options sound quite self-explanatory.